src/xb-builder.c

```diff
@@ -648,11 +648,15 @@ xb_builder_watch_source (XbBuilder *self,
 {
 	XbBuilderPrivate *priv = GET_PRIVATE (self);
 	GFile *file = xb_builder_source_get_file (source);
+	g_autoptr(GFile) to_watch = NULL;
```

Comment:
maybe `file_monitor`?

Reply:
`file_monitor` makes it sound like it’s a `GFileMonitor`. I’ve changed it to `watched_file`, how’s that?